lib/template.html

```diff
@@ -2,22 +2,34 @@
 <html>
 <head>
     <title>Auditly</title>
+    <link rel="stylesheet" type="text/css" href="./style.css">
+
 </head>
-<body style="background-color:black">
+<body>
     <center>
-        <H style="color:#ffec80; font-size:450%;font-family:Verdana;"><strong>Auditly CVE Report<strong></H1>
+        <H class="head"><strong>Auditly CVE Report<strong></H1>
```

Comment:
use the `<h1>` tag and that will get you most of the way to where you want to be with this one without much additional styling.